ranger-plugin/conf/rps.properties

```diff
@@ -29,4 +29,7 @@ RPS_SHUTDOWN_PORT=8405
 RANGER_HAWQ_INSTANCE=hawq
 
 # version of the software
-RPS_VERSION=${project.version}
\ No newline at end of file
+RPS_VERSION=${project.version}
+
+# home path of Tomcat/6.0.x for Ranger Plugin Service
+CATALINA_HOME=
```

Comment:
should this line not be here by default and only be added by users configuring the system from the build script ? Users that install from RPM do not need this property and might be confused why it is empty in this file.